### ISSUE TITLE
Deploy resources error - would benifit from better error message

### ISSUE THREAD

**Issue by OneMuppet** [serverless/serverless#233]
This is the jaws output:
´´´javascript
  CreationTime: Wed Oct 14 2015 18:28:21 GMT+0200 (CEST),
  LastUpdatedTime: Thu Oct 15 2015 14:50:29 GMT+0200 (CEST),
  StackStatus: 'UPDATE_ROLLBACK_IN_PROGRESS',
  StackStatusReason: 'The following resource(s) failed to create: [DynamoDBShipmentsTable]. ',
  DisableRollback: false,
  NotificationARNs: [],
  Capabilities: [ 'CAPABILITY_IAM' ],
  Outputs: [],
  Tags: [ { Key: 'STAGE', Value: 'dev' } ] }
JAWS: Resources Deployer  "dev - eu-west-1":  Cloudformation stack update failed because of the following error...  
{ [JawsError: Something went wrong while updateing your cloudformation]
  name: 'JawsError',
  message: 'Something went wrong while updateing your cloudformation',
  messageId: undefined }
´´´
but when looking in the API Console for cloudformation I see this:
![image](https://cloud.githubusercontent.com/assets/4235764/10514348/7f8109bc-734d-11e5-9467-78063142c165.png)

Is it possible to add more details to the error messages?

**Comment by apaatsio** [comment]
@OneMuppet Could you provide steps to reproduce this error? Thanks!

**Comment by dennisnewel** [comment]
i just got the same error message. What i did, that may or may not have caused it, was to

* create two new modules
* manually rename folders and references to rename the modules
* try to deploy the new modules after renaming

i'm still trying to figure out if the manual renaming could have caused the issue. I didn't deploy before renaming

**Comment by eahefnawy** [comment]
I think this should be fixed now that we released Serverless. Closing now and feel free to reopen if this is still an issue.

Thanks tons guys!
